src/main/proto/eventtemplates/other.proto

```diff
@@ -14,31 +14,17 @@
 
 syntax = "proto3";
 
-package eventtemplates;
+package halo_cmm.uk.pilot;
 
 import "wfa/measurement/api/v2alpha/event_annotations.proto";
 
-option java_package = "eventtemplates";
+option java_package = "org.halo_cmm.uk.pilot";
 option java_multiple_files = true;
 
-message VideoEventTemplate {
+message Other {
```

Comment:
+1 to Ugur's comment, I'd suggest to align naming, eg call this "OtherMediaEvent"

Same applies to the file name